contrib/macdeploy/macdeployqtplus

```diff
@@ -302,7 +302,6 @@ def copyFramework(framework, path, verbose):
         if os.path.exists(fromContentsDir):
             toContentsDir = os.path.join(path, framework.destinationVersionContentsDirectory)
             shutil.copytree(fromContentsDir, toContentsDir, symlinks=True)
-            contentslinkfrom = os.path.join(path, framework.destinationContentsDirectory)
```

Comment:
In af0bd5e, the majority of this code was removed but I think this line was left in by mistake. As far as I can tell it doesn't do anything - it just defines a variable that is never used.